### ISSUE TITLE
Converting FP16 ONNX model to TOSA MLIR?

### ISSUE THREAD

**Issue by hashimSharif** [onnx/onnx-mlir#1805]
Hi, 

Does this project support exporting FP16 models to TOSA dialect? I am assuming from the `isTOSAFloat` implementation here https://github.com/onnx/onnx-mlir/blob/main/src/Conversion/ONNXToTOSA/ONNXToTOSACommon.hpp#L49 that FP16 is supported?

Any pointers for exporting FP16 onnx models is appreciated.

-Hashim

**Comment by philass** [comment]
Hey @hashimSharif, 

I think the TOSA export path isn't fully implemented. If I am not mistaken there is only really a a skeleton for an implementation.

I don't think there is a lot of active work on the TOSA path at the moment.

**Comment by AlexandreEichenberger** [comment]
Yes, please let  us know if there is further interest in this path, or if maybe other paths out of ONNX dialects are subsuming this effort.

Thanks for your input on this.
